tests/robot-tests/tests/libs/create_emulator_release_files.py

```diff
@@ -64,7 +64,9 @@ def __init__(self):
 
         # Instantiate a new ContainerClient for the emulator
         self.blob_service_client = BlobServiceClient.from_connection_string(
-            "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://data-storage:10000/devstoreaccount1")
+            "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey"
+            "=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint"
+            "=http://data-storage:10000/devstoreaccount1")
```

Comment:
Could format this to be a little nicer by pairing they keys up with the values ie:

```py
"DefaultEndpointsProtocol=http;"
"AccountName=devstoreaccount1;"
"AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;"
"BlobEndpoint=http://data-storage:10000/devstoreaccount1"
```

Reply:
Looks like the storage emulator doesn't like it when we split the connection string up into separate strings: 

```
binascii.Error: Invalid base64-encoded string: number of data characters (85) cannot be 1 more than a multiple of 4
```

Reply:
Can't we just concatenate the strings using `+` then?